markovflow/kernels/sde_kernel.py

```diff
@@ -390,6 +390,15 @@ def initial_mean(self, batch_shape: tf.TensorShape) -> tf.Tensor:
         shape = tf.concat([tf.TensorShape(batch_shape), self._state_mean.shape], axis=0)
         return tf.broadcast_to(self._state_mean, shape)
 
+    def set_state_mean(self, state_mean: tf.Tensor, trainable: bool = False):
```

Comment:
would you mind updating the notebook which actually set the state mean as part of this pr?
I think it is the piece_wise kernel one
or maybe the choosing and combining kernel (or both)